test/test_autograd.py

```diff
@@ -6039,6 +6041,246 @@ def run_functional_checks(test_case, test_name, name, apply_fn, run_grad_checks,
         test_case.assertEqual(self_variable.size(), self_variable.grad.size())
 
 
+# borrowed from test/test_cpp_extensions_jit.py
```

Comment:
put this in a common helper file?

Reply:
or, I'd actually suggest putting these tests in one of the extensions file, so that people can continue to run autograd tests without having to worry about compiling cpp extensions